src/world/area_hos/hos_05/A2AAC0.c

```diff
@@ -997,7 +1045,52 @@ ApiStatus func_802444E8_A2E728(Evt* script, s32 isInitialCall) {
     return ApiStatus_BLOCK;
 }
 
+#ifdef NON_MATCHING
+ApiStatus func_80244550_A2E790(Evt* script, s32 isInitialCall) {
+    Npc* npc = resolve_npc(script, 9);
```

Comment:
There should be an npc enum in the map, can you use that instead of `9`?

Reply:
It hasn't been set up yet - clover hasn't gotten to this one. the other funcs here also use 9

Reply:
I'll handle the cleanup for this when I get to this map. I'm pretty thorough with it; I read through each function/script line by line as I do it looking for stuff like this.